packages/commonwealth/client/scripts/views/components/Mava/Mava.tsx

```diff
@@ -0,0 +1,49 @@
+import { memo, useCallback, useEffect, useRef } from 'react';
+import app from 'state';
+import { isWindowSmall } from 'views/components/component_kit/helpers';
+import './Mava.scss';
+
+// eslint-disable-next-line no-var
+declare var window: any;
```

Comment:
code quality checks is not passing because of this

Reply:
Ha.  Thanks. I guess I didn't see this because of all the other read alerts going off with our other failed tests.